### ISSUE TITLE
Questions for finishing the best fusion reactor project yet

### ISSUE THREAD

**Issue by snuufix** [ReikaKalseki/Reika_Mods_Issues#1068]
Hello Reika. First of all, I've been a silent fan of your work for some years now. Thank you so much for creating things that replicate latest science stuff so closely. If something is very hard to create even in creative, it speaks for itself about the quality of your mods.

I'm building my third fusion reactor and this time it will power whole server, so it must be perfect enough to create majestic amounts of power, while being mostly maintenance free. But my goals also include making it without creative/spawned supplies, [beautiful](https://imgur.com/a/0xp6p) and mostly realistic. (I dislike teleporting liquids & stuff, so there are LOTS of organized piping) I've been passively building it for about a year now and I've achieved most of the goals, except 100% automation. The reactor already have closed cycle ammonia cooling and it will detect failing absorbers/boilers, it even have emergency battery with relay on low power, seperated from main storage and even a backup turbine, so it's mostly without 'single point of failure' problems. But the thing is, it generates ungodly amounts of power that rarely will be fully utilized and it would be nice to turn reactor off compleatly without manual intervention. It can already automatically manage most of liquid production, act on power failures and spin down/up individual turbines depending on situation, with only two exceptions: preheaters and injectors can be controlled only manually. The problem is that I can't get readings about how much steam or plasma is already produced in the pipes, so I can't automate reactor shutdown operations and if injectors are enabled, without running turbines giving back low pressure ammonia, the closed loop supply of pressurized ammonia gets consumed pretty fast and reactor shuts down because it detects that it needs manual fixing. (well, better than exploding boilers but still...)

TL;DR:
- How can I measure steam in steam pipes and plasma in containment pipes? They reject OpenComputers/CC/peripheral adapters. This is preventing me to fully automate the reactor and making it truly energy efficient.

Also, while at it, is there a better way to shut down high pressure turbine than cutting lubricant supply? I'm afraid that running without lubricant could damage the turbines in ReactorCraft future releases. Steam pumps doesn't distribute steam to turbines evenly, so I can't use them to cut off steam supply.

**Comment by ReikaKalseki** [comment]
Redstone can shut down an HP turbine (apply it to the injectors), and I have been told that adapters _do_ work with steam lines.

**Comment by redelman** [comment]
At least the last time I tried it, back in v13, adapters worked fine on the steam lines. I haven't played at all since about v13a though.

**Comment by zemerick** [comment]
As for the injectors and preheater:

Injector can be controlled with redstone under the injector block.

Preheater you should only have 1 ( I doubt you have more than 50 injectors ), and it really shouldn't be turned off due to the massive warm up time.

Also for the future, you should really post questions and such to the forums. The issue tracker is meant for actual problems in the mods.
